Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/zipline .
+# 
+# To run the container:
+#
+#    docker run -v=/path/to/your/notebooks:/projects -p 8888:8888/tcp --name zipline -it quantopian/zipline
```

Comment:
Hrm...and now I can't reproduce it at all.  My best guess is there are some funky browser cache interactions happening here.  Probably not worth digging into unless you're really excited about that sort of thing...